Formula 1/ViewModels/CircuitsViewModel.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  CircuitsViewModel.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+
+class CircuitsViewModel: ObservableObject {
+    // using @Published for when implementing with SwiftUI
+    @Published private var circuits = [Circuit]()
```

Comment:
Either comment the line or remove private.

Reply:
I keep as private because I have accessors